src/System/Wlog/Launcher.hs

```diff
@@ -124,3 +127,42 @@ buildAndSetupYamlLogging configBuilder loggerConfigPath = do
 -- See this module description.
 initLoggingFromYaml :: MonadIO m => FilePath -> m ()
 initLoggingFromYaml = buildAndSetupYamlLogging mempty
+
+-- | Initializes logging using given 'FilePath' to logger configurations,
+-- runs the action with the given 'LoggerName'.
+runLoggingWithFile :: MonadIO m => FilePath -> LoggerName -> LoggerNameBox m a -> m a
+runLoggingWithFile filename logName f = do
+    buildAndSetupYamlLogging productionB filename
+    usingLoggerName logName f
+
+-- | Default logging configuration with the "node" logger.
+defaultConfig :: LoggerConfig
```

Comment:
Could you please create `defaultConfig` using lenses? Unfortunately, this feature is not specified anywhere, so there was no chance for you to know about it :( But creating `LoggerConfig`s manually is the thing that supposed to be done quite often. So I invented convenient DSL for such logging initialization.

You can see this feature here:
1. https://github.com/serokell/log-warper/blob/5783fde3f8cca225e146d08173584dacfe67ed3b/test/Test/Wlog/RollingSpec.hs#L66
2. https://github.com/input-output-hk/cardano-sl/blob/8507d03ba928e07daea57f9a52f6dc03a9d65779/lib/src/Pos/Client/CLI/Util.hs#L61
3. https://issues.serokell.io/issue/LW-12

If you have any questions, don't be shy and ask them :)